Sudoku/sudoku.spec.js

```diff
@@ -27,78 +27,78 @@ describe('Sudoku solver', function() {
   });
 
   //This puzzle cannot be solved, because the middle column (c5) has no possible candidates for the value ‘2’.
-  it('Detects an unsolvable column', function() {
+  xit('Detects an unsolvable column', function() {
     var sudoku = new Sudoku("....41....6.....2...2......32.6.........5..417.......2......23..48......5.1..2...");
     expect(sudoku.solve()).toBe("ERROR: Bad Board");
   });
 
   // This puzzle cannot be solved, because the middle row (r5) has no possible candidates for the value ‘1’.
-  it('Detects an unsolvable row', function() {
+  xit('Detects an unsolvable row', function() {
     var sudoku = new Sudoku("9..1....4.14.3.8....3....9....7.8..18....3..........3..21....7...9.4.5..5...16..3");
     expect(sudoku.solve()).toBe("ERROR: Bad Board");
   });
 
-  it('recognizes a solved board as solved', function() {
+  xit('recognizes a solved board as solved', function() {
```

Comment:
The `isSolved` function is empty. How does this test work with an empty function? Did you write the tests first and then plan on implementing functionality? I can appreciate that if that's the case. :)